sdk/clientcore/annotation-processor/src/main/java/io/clientcore/tools/codegen/utils/ResponseBodyModeGeneration.java

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.tools.codegen.utils;
```

Comment:
nit: update package names for this lib too to follow the group + artifact name convention.